web/src/lib/components/shared-components/immich-thumbnail.svelte

```diff
@@ -149,8 +154,6 @@
 
 <IntersectionObserver once={false} let:intersecting>
 	<div
-		style:width={`${thumbnailSize}px`}
-		style:height={`${thumbnailSize}px`}
```

Comment:
Is there a reason we had that? AFAIK this information was already provided by the CSS class?

Reply:
This probably doesn't work. Class names shouldn't contain dynamic values, use style instead